src/main/java/com/o19s/es/ltr/logging/LoggingFetchSubPhase.java

```diff
@@ -74,54 +74,81 @@ public void hitsExecute(SearchContext context, SearchHit[] hits) throws IOExcept
             });
         }
 
-        doLog(builder.build(), loggers, context.searcher(), hits);
-
+        return new FetchSubPhaseProcessor() {
+            @Override
+            public void setNextReader(LeafReaderContext readerContext) {
+            }
+           
+            @Override
+            public void process(HitContext hitContext) throws IOException {
+                doLog(builder.build(), loggers, context.searcher(), hitContext.hit());
+            }
+        };
     }
+    // public void hitsExecute(SearchContext context, SearchHit[] hits) throws IOException {
```

Comment:
This should be ripped out before merge, but left as an artifact of the way it was before v7.10